bin/loopback-cli.js

```diff
@@ -36,10 +36,10 @@ process.env.SLC_COMMAND = 'loopback-cli';
 // therefore I am intentionally loading it only after we have
 // handled the "--version" case which becomes much faster as the result.
 const lbGenerator = require('generator-loopback');
-const yeoman = lbGenerator._yeoman; // generator-loopback should export _yeoman
-assert(yeoman, 'generator-loopback should export _yeoman');
+const yeomanEnv = lbGenerator._yeomanEnv; // generator-loopback should export _yeomanEnv
```

Comment:
nitpick: I don't think this comment is needed considering the assert on the next line